tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -12,6 +14,39 @@
 
 
 class TestAESService:
+    @freeze_time("2022-1-15")
+    def test_rotate(self, svc_generates, valid_keys, expired_keys, db_session):
+        """Keys created the 10th, max ages of 2 and 4 both expire valid and deleted expired on the 15th."""
+        target_keys = 5
+
+        # Assert what we think is the initial setup
+        assert not any((key.expired for key in valid_keys))
+        assert all((key.expired for key in expired_keys))
+        assert all((key in db_session for key in expired_keys))
```

Comment:
```suggestion
        # Assert what we think is the initial setup
        assert not any((key.expired for key in valid_keys))
        assert all((key.expired for key in expired_keys))
        assert all((key in db_session for key in expired_keys))
        
        target_keys = 5
```

Keeps the var nearer where it's used